app/resources/v1/streets_pg.js

```diff
@@ -65,8 +60,8 @@ exports.post = async function (req, res) {
     }
   }
 
-  function updateUserLastStreetId (userId) {
-    return User.findByPk(userId).then((user) => {
+  function updateUserLastStreetId (userAltId) {
+    return User.findOne({ where: { _id: userAltId } }).then((user) => {
```

Comment:
Let's try to do this without using the `_id` field. See comment above regarding where this comes from.